Websites/webkit.org/wp-content/themes/webkit/standards-positions.php

```diff
@@ -0,0 +1,176 @@
+    } catch(e) {}
+  }
+  return null; 
+}
+
+function createStandardsPositionsTableRows(issues, filterText = "", filterTopic = "") {
+  const fragment = new DocumentFragment();
+  const filterTextRE = createFilterTextRE(filterText);
+  for (const issue of issues) {
+    if (filterTextRE && issue["title"].search(filterTextRE) === -1) {
+      continue;
+    }
+    if (filterTopic && !issue["topics"].includes(filterTopic)) {
+      continue;
+    }
+    const id = issue["id"].substring("https://github.com/WebKit/standards-positions/issues/".length);
+    
+    const row = addElement(fragment, "tr", { id: `position-${id}` });
+    addElement(row, "td", {}, createElement("a", { href:issue["id"] }, `#${id}`));
+    if (issue["url"] !== null) {
+      addElement(row, "td", {}, createElement("a", { href:issue["url"] }, issue["title"]));
+    } else {
+      addElement(row, "td", {}, issue["title"]);
+    }
+    addElement(row, "td", {}, issue["concerns"].join(", "));
+    if (issue["position"] !== null) {
+      addElement(row, "td", { class: issue["position"].replace(" ", "-") }, createElement("span", {}, issue["position"]));  
+    } else {
+      addElement(row, "td");      
+    }
+    addElement(row, "td", {}, issue["topics"].join(", "));
+    addElement(row, "td", {}, issue["venues"].join(", "));
+    addElement(row, "td", {}, createElement("a", { href: `#position-${id}`, title: `Link to position: ${issue["title"]}` }, "⚓︎"));
+  }
+  return fragment;
+}
+
+let standardsPositions = null;
+let tbody = null;
+const container = document.querySelector("main"); 
+const noScriptChild = container.querySelector("p");
+const loadingChild = createElement("p", {}, "Please hold while we fetch data from GitHub.");
+noScriptChild.replaceWith(loadingChild);
+
+function updateStandardsPositionsTableRows() {
+  const filterText = document.getElementById("filter").value;
+  const filterTopic = document.getElementById("filter-topic").value;
+  tbody.replaceChildren(createStandardsPositionsTableRows(standardsPositions, filterText, filterTopic));
+}
+
+function updateTopicFilter() {
+  const topics = [];
+  for (const issue of standardsPositions) {
+    for (const topic of issue["topics"]) {
+      if (!topics.includes(topic)) {
+        topics.push(topic);
+      }
+    }
+  }
+  topics.sort();
+  const select = document.getElementById("filter-topic");
+  addElement(select, "option", { value:"" }, "all");
+  for (const topic of topics) {
+    addElement(select, "option", {}, topic);
+  }
+}
+
+fetch("https://raw.githubusercontent.com/WebKit/standards-positions/main/summary.json").then(res => res.json()).then(positions => {
+  loadingChild.replaceWith(createStandardsPositionsTable());
+  tbody = container.querySelector("tbody");
+  standardsPositions = positions;
+  updateTopicFilter();
+  updateStandardsPositionsTableRows();
+}).catch(() => {
+  loadingChild.replaceWith(createElement("p", {}, "Something went wrong. Please try again later."));
+});
```

Comment:
Just for us in case something does start going wrong. 

```suggestion
}).catch((err) => {
  loadingChild.replaceWith(createElement("p", {}, "Something went wrong. Please try again later."));
  console.error(err);
});
```